src/gaurav_garg/PatternPrintAssignment78.java

```diff
@@ -0,0 +1,51 @@
+/*
+ Pattern -: 
+*
+**
+***
+****
+
+1
+1 2
+1 2 3 
+1 2 3 4
+1 2 3 4 5
+1 2 3 4 5 6
+
+1
+2 3
+4 5 6
+7 8 9 10
+ */
+
+package gaurav_garg;
+
+public class PatternPrintAssignment78 {
+	public static void main(String[] args) {
+
+		for (int i = 0; i < 4; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible